### PR TITLE
CNV-36165: Release note: NEW

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.15

Issue: [CNV-36165](https://issues.redhat.com/browse/CNV-36165)

Link to docs preview: 4.15 release notes, see note under [Web console](https://71593--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-web) section

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
Hi @upalatucci, Can you please review this release note? Thanks!

**Review by upalatucci — APPROVED** [review]

**Comment by xenolinux** [comment]
LGTM
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -107,7 +107,12 @@ This release adds new features and enhancements related to the following compone
 
 //CNV-33122: Virtualization page updates for 4.15
 
-//CNV-36165: VM disks with persistent reservation
+//CNV-36165: VM disks with persistent reservation. from Audrey: I will add xrefs to this note when the adoc files have been merged.
+* You can configure disk sharing for ordinary VM or LUN-backed VM disks to allow multiple VMs to share the same underlying storage. Any disk to be shared must be in block mode.
++
+To allow a LUN-backed block mode VM disk to be shared among multiple virtual machines, a cluster administrator must enable the SCSI `persistentReservation` feature gate.
```
Comment (adellape) [inline virt/release_notes/virt-4-15-release-notes.adoc:113]:
```suggestion
* You can configure disk sharing for ordinary virtual machine (VM) or LUN-backed VM disks to allow multiple VMs to share the same underlying storage. Any disk to be shared must be in block mode.
+
To allow a LUN-backed block mode VM disk to be shared among multiple VMs, a cluster administrator must enable the SCSI `persistentReservation` feature gate.
```

Comment (aspauldi) [reply virt/release_notes/virt-4-15-release-notes.adoc]: corrected

**Review by adellape — APPROVED** [review]

**Pull request merged**